lib/screens/episode.dart

```diff
@@ -251,71 +248,17 @@ class _EpisodeScreenState extends ConsumerState<EpisodeScreen> with AutoRouteAwa
     final player = ref.watch(playerProvider);
     final currentPosSeconds = ((player?.playbackPositionMs ?? 0) / 1000).round();
     final formattedPosition = getFormattedDuration(currentPosSeconds, padFirstSegment: true);
```

Comment:
Passing in the formattedPosition is unecessary. Pass only the currentPosSeconds, and format it inside the ShareEpisodeSheet widget instead.